odoo/addons/base/res/res_bank.py

```diff
@@ -32,6 +33,15 @@ class Bank(models.Model):
     active = fields.Boolean(default=True)
     bic = fields.Char('Bank Identifier Code', index=True, help="Sometimes called BIC or Swift.")
 
+    @api.model
+    def _fields_view_get(self, view_id=None, view_type='form', toolbar=False, submenu=False):
+        if (not view_id) and (view_type == 'form'):
+            view_id = self.env.ref('base.view_res_bank_form').id
+        res = super(Bank, self)._fields_view_get(view_id=view_id, view_type=view_type, toolbar=toolbar, submenu=submenu)
+        if view_type == 'form':
+            res['arch'] = self._fields_view_get_address(res['arch'])
```

Comment:
@eternalharvest I think you should probably define another method (e.g. _fields_view_get_address_bank()) instead of using _fields_view_get_address(), and perhaps a place for configuring the layout.  IMO, though, adjustments to company and bank layouts can be an overkill as these layouts are usually not really the pain points of the users as far as I'm aware.